lib/omni_auth/strategies/open_id_connect_decorator.rb

```diff
@@ -0,0 +1,143 @@
+module OmniAuth
```

Comment:
The work authorization is specific to ReShare but other aspects might be relevant to Hyku.  Let's hold on moving this back to Hyku.